addon/components/tooltip-on-element.js

```diff
@@ -67,8 +67,11 @@ export default EmberTetherComponent.extend({
   'aria-hidden': computed.not('tooltipIsVisible'),
   attributeBindings: ['aria-hidden', 'role', 'tabindex'],
   classNameBindings: ['effectClass'],
-  classPrefix: 'tooltip',
-  classNames: ['tooltip'],
+  classPrefix: 'ember-tooltip',
+  classNames: ['ember-tooltip', 'tooltip'],
```

Comment:
Thanks Miguel.

@sir-dunxalot I think we do run a risk of existing projects having already applied styles to `tooltip-target-attached-left`, although the risk of that is much smaller than the risk of them having applied styles to `.tooltip` (which we do have temporary backwards compatibility for). The question becomes do we want to support backwards compatibility for the edge-cases of `tooltip-*` classes.

I think we should add release notes about continuing, temporary support for the `tooltip` class but say that the `tooltip-*` classes will be deprecated in the release and should be addressed before upgrading.